terraform/stacks/unimelb/data_archive/analysis_archive.tf

```diff
@@ -0,0 +1,101 @@
+################################################################################
+# Local constants
+
+locals {
+  # The bucket holding all archived analysis data
+  analysis_archive_bucket_name = "archive-prod-analysis-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}"
```

Comment:
This bucket will need to be configured as an event source in orcabus too, if filemanager wants to ingest events?

Reply:
Yes I forgot the event bus config... adding...

Reply:
Added with https://github.com/umccr/infrastructure/pull/490/commits/d2ec42bb2ae8073eb2c0ba0da3a18ebf77a1d3b4

Note: this does not include the activation of the eventbridge integration at the bucket level (due to possible integration with ICA BYOB). Any subscriber (like the FileManager) will have to set up their own rules against the OrcaBus event bus.

Reply:
Is the intent to have this contain just prod data? E.g. there won't be an `archive-dev-analysis-*` bucket?

Reply:
Yes, we don't have plans to archive any `dev` data. That's a `prod` thing only.